.idea/StowarzyszenieNaukiJavy.iml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Katalog `.idea` powinien chyba znaleźć się w `.gitignore` żeby nie wrzucać go do zdalnego repo

Reply:
Zgadzam się, ale miałem jakieś problemy z tym, spróbuje to naprawić.

Reply:
Wystarczy chyba dodać takie 2 linijki:
```
*.iml
.idea
```
 to wytnie Ci wszystkie pliki z rozszerzeniem `.iml` i cały katalog `.idea`